docs/source/public/input_preparation.rst

```diff
@@ -140,6 +140,30 @@ If the script outputs “SIMULATION COMPLETE”, then your inputs are suitable f
 .. note::
    This script runs a very short simulation, it is recommended that it is executed on a machine with a CUDA-enabled GPU.
 
+Preparing the ligand file
+=========================
+
+For some datasets, the Schrodinger public binding free energy benchmark set includes multiple binding modes (e.g. different rotamers) 
+and protonation states for some of the ligands. For this current study, we will only consider a single conformation and protonation state of the ligands. 
+Here, we will be using the binding mode and protonation state that was predicted to be most potent by FEP+.
+
+If the dataset contains ligands in multiple conformations or protonation states, the more favorable state should be identified (by looking at the results deposited by Schrodinger) and the less favorable state should be removed from the input ``ligands.sdf`` file.
+
+**Assessing the more potent binding mode or protonation state** 
+
+The FEP+ edge predictions can be found `here <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/main/21_4_results/edge_predictions>`_. 
```

Comment:
Just checking - we're using the edge predictions because the ligand predictions don't include all the alternates right?

Do we know if the ligand predictions are just already the best binders? (in which case we could just say "look at the ones listed here, these are the ones you want")

Reply:
Following up from Friday's discussions - we need to look at the dG ligand results for the protonation states. This might influence what we advise here (or it'll just make these instructions more complicated).

Reply:
I changes the instructions to look at the ligand predictions instead. Missing currently are the instruction for the case of different protonation states, will update that after the meeting.